Marlin/src/inc/Conditionals_adv.h

```diff
@@ -547,82 +547,3 @@
   #define USING_SERIAL_8 1
 #endif
 #undef ANY_SERIAL_IS
-
-// Custom user menus
-#define HAS_CUSTOM_USER_MENU_ITEM(N) (defined(CUSTOM_USER_MENUS) && defined(USER_DESC_##N) && defined(USER_GCODE_##N))
```

Comment:
I just follow the same pattern that is used right above, for `ANY_SERIAL_IS` and removed duplicated code.

Reply:
Those are pretty well obsolete now too. Removing….